kura/test/org.eclipse.kura.core.testutil/src/main/java/org/eclipse/kura/core/testutil/requesthandler/RestTransport.java

```diff
@@ -87,6 +97,21 @@ public void init() {
             throw new IllegalStateException(e);
         }
     }
+    
+    private <T> T trackService(final Class<T> classz)
+            throws InterruptedException, ExecutionException, TimeoutException {
+        return ServiceUtil
+                .trackService(classz, Optional.empty()).get(30, TimeUnit.SECONDS);
+    }
+    
+    private Map<String, Object> initialRestServiceConfiguration() throws KuraException {
+
+        final Map<String, Object> restServiceConfiguration = new HashMap<>();
+
+        restServiceConfiguration.put("allowed.ports", "[8080,443,4443]");
```

Comment:
Shouldn't `allowed.ports` value be an array of integers instead of a string?

Reply:
If the idea here is to restore the previous behaviour we could also specify an empty integer array to remove all restrictions on allowed ports